### DISCUSSION TITLE
绳匠们我挂个人（盒武器）

### DISCUSSION THREAD

**Discussion by Skadi11777** [share121/inter-knot#172] [其他]
![1722955385309](https://github.com/user-attachments/assets/0ca7168f-a4c7-4004-b0d2-91244dddb8a3)
绳匠们我挂个人哈

**Comment by gjbctuj** [comment]
我超，盒！

**Comment by Liangjl-2024** [comment]
爱丽丝错了，爱丽丝不应该在网上口嗨的～

**Comment by Goto-Hitori** [comment]
是千年的爱丽丝同学吗

**Comment by myt521** [comment]
我超，盒！

**Comment by koyki233** [comment]
狙了

**Comment by Lckaer** [comment]
美味小孩，我来啦![05DC8906](https://github.com/user-attachments/assets/45139945-97cf-4f91-b48e-2aaaf868679c)
![05DC8BC5](https://github.com/user-attachments/assets/84b39430-e947-4178-ab35-db5c45419627)

**Comment by shishenyihe** [comment]
这个委托我接了

**Comment by moonhalo7** [comment]
↑↓啊
